src/main/main.cpp

```diff
@@ -302,7 +313,7 @@ main(int argc, char* const* argv)
         Logging::setLogLevel(logLevel, nullptr);
 
         cfg.REBUILD_DB = newDB;
-        cfg.FORCE_SCP = forceSCP;
+        cfg.FORCE_SCP = (forceSCP != nullptr);
```

Comment:
yeah I agree we don't need to set cfg.FORCE_SCP here. The old code was also wrong